.travis.yml

```diff
@@ -1,4 +1,7 @@
 language: node_js
+os:
+    - linux
+    - osx
```

Comment:
no windows test? Also these changes are unrelated to this branch (leave them cause they are small though)